doc/samtools-cat.1

```diff
@@ -122,8 +122,6 @@ container but outside of the requested region.
 .BI --no-PG
 Do not add a @PG line to the header of the output file.
 .TP
```

Comment:
This line needs to be removed as well, else the "EXAMPLES" header underneath ends up in the wrong place.

Reply:
thanks, the issue is visible in linux and now the indentation is removed.